rotkehlchen/chain/manager.py

```diff
@@ -545,6 +545,15 @@ def get_module(self, module_name: ModuleName) -> Optional[Any]:
         return instance
 
     def get_balances_update(self) -> BlockchainBalancesUpdate:
+        db_settings = self.database.get_settings(have_premium=bool(self.premium))
+        balances = self.balances
+        if db_settings.eth_equivalent_eth2 and balances.eth2 is not None:
+            new_balances: defaultdict[Union[Eth2PubKey, ChecksumEthAddress], BalanceSheet] = (
+                defaultdict(BalanceSheet, balances.eth2 | balances.eth)  # type: ignore
+            )
+            balances.eth = new_balances
+            balances.eth2.clear()
```

Comment:
I'm confused as to what you mean by modifying `A_ETH2 = A_ETH`? Where should I do that, as in I can keep the logic, but where should I keep it so that it's replaced each time the server is run.

Reply:
>  in the settings modification of the new setting